jenkins-scripts/dsl/ignition_collection.dsl

```diff
@@ -343,26 +343,26 @@ ignition_collections.each { ign_collection ->
 
   // MAC Brew CI job
   // --------------------------------------------------------------
-  def ignition_brew_ci_job = job("ignition_${ign_collection_name}-ci-main-homebrew-amd64")
+  def ignition_brew_ci_job = job("ignition_${gz_collection_name}-ci-main-homebrew-amd64")
   OSRFBrewCompilation.create(ignition_brew_ci_job, DISABLE_TESTS)
   OSRFGitHub.create(ignition_brew_ci_job,
-                    "gazebosim/ign-${ign_collection_name}",
+                    "gazebosim/ign-${gz_collection_name}",
                     "main",
-                    "ign-${ign_collection_name}")
+                    "ign-${gz_collection_name}")
   ignition_brew_ci_job.with
   {
       steps {
         shell("""\
               #!/bin/bash -xe
 
-              /bin/bash -xe "./scripts/jenkins-scripts/lib/project-default-devel-homebrew-amd64.bash" "ignition-${ign_collection_name}"
+              /bin/bash -xe "./scripts/jenkins-scripts/lib/project-default-devel-homebrew-amd64.bash" "gz-${gz_collection_name}"
```

Comment:
Could become `gz-gazebo`

Reply:
Hmmmmmm..... If I change it the other way it could become ignition-sim :/

Let me see if I can do some logic